modules/sdk-core/src/bitgo/utils/tss/ecdsa/ecdsa.ts

```diff
@@ -697,8 +697,8 @@ export class EcdsaUtils extends baseTSSUtils<KeyShare> {
       encryptedSignerShare,
       vssProof,
       privateShareProof,
-      userPublicGpgKey,
-      publicShare
+      publicShare,
+      userPublicGpgKey
```

Comment:
Can you add a testcase to test this fix?,

Reply:
The API will support both cases to be backwards compatible with the previous SDK version that was already released.